editoast/openapi.yaml

```
@@ -1324,6 +1324,39 @@ paths:
                         train_path:
                           type: number
 
+  /timetable/{id}/conflicts/:
+    get:
+      tags:
+        - timetable
+      summary: Retrieve all conflicts for a specific timetable
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          description: Timetable ID
+          required: true
+      responses:
+        200:
+          description: The timetable conflicts content
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    trains_id:
+                      type: array
+                      items:
+                        type: number
+                    start_time:
+                      type: string
+                    end_time:
+                      type: string
+                    conflict_type:
+                      type: string                 
```

```suggestion
                      type: string
                  required: [trains_id, start_time, end_time, conflict_type]
```
I suppose these fields are required :)

---

Done !